api/python/provisioner/salt.py

```diff
@@ -798,6 +809,7 @@ def _salt_runner_cmd(  # noqa: C901 FIXME
     fun_args: Union[Tuple, None] = None,
     fun_kwargs: Union[Dict, None] = None,
     nowait=False,
+    secure=False,
```

Comment:
Should the default value of secure not be True?

Reply:
No, it would impact many places where we don't need that.